

make `make allmodconfig' work.  We need at least one statically-linked IO
scheduler.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/Kconfig.iosched |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/block/Kconfig.iosched~switchable-and-modular-io-schedulers-hack-fix drivers/block/Kconfig.iosched
--- 25/drivers/block/Kconfig.iosched~switchable-and-modular-io-schedulers-hack-fix	2004-10-01 20:36:36.027164528 -0700
+++ 25-akpm/drivers/block/Kconfig.iosched	2004-10-01 20:38:41.210133824 -0700
@@ -1,5 +1,5 @@
 config IOSCHED_NOOP
-	tristate "No-op I/O scheduler"
+	bool
 	default y
 	---help---
 	  The no-op I/O scheduler is a minimal scheduler that does basic merging
_
